lighthouse-core/audits/seo/link-text.js

```diff
@@ -63,7 +63,7 @@ const UIStrings = {
   failureTitle: 'Links do not have descriptive text',
   /** Description of a Lighthouse audit that tells the user *why* they need to have descriptive text on the links in their page. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'Descriptive link text helps search engines understand your content. ' +
-  '[Learn more](https://developers.google.com/web/tools/lighthouse/audits/descriptive-link-text).',
+  '[Learn more](https://web.dev/link-text).',
```

Comment:
FYI the screenshot in this doc is of a Lighthouse error/bug instead of the audit failing